internal/extproc/translator/openai_awsbedrock.go

```diff
@@ -646,6 +680,9 @@ func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) convertEvent(event *awsbe
 			PromptTokens:     event.Usage.InputTokens,
 			CompletionTokens: event.Usage.OutputTokens,
 		}
+		o.metrics.TokensTotal.WithLabelValues(o.modelName, "total").Add(float64(event.Usage.TotalTokens))
+		o.metrics.TokensTotal.WithLabelValues(o.modelName, "prompt").Add(float64(event.Usage.InputTokens))
+		o.metrics.TokensTotal.WithLabelValues(o.modelName, "completion").Add(float64(event.Usage.OutputTokens))
```

Comment:
Aren't these metrics already recorded by the processor?

Reply:
all the metrics are now encapsulated.